v2/src/ctrlf_auth/views.py

```diff
@@ -53,3 +53,18 @@ def post(self, request, *args, **kwargs):
             for _, message in serializer.errors.items():
                 message = message[0]
         return Response(data={"message": message}, status=status.HTTP_400_BAD_REQUEST)
+
+
+class CheckNicknameDuplicateView(APIView):
+    _SUCCESS_MSG = "사용 가능한 닉네임입니다."
+
+    @swagger_auto_schema(query_serializer=NicknameDuplicateSerializer)
+    def get(self, request):
+        nickname = request.GET
```

Comment:
`request.GET` 보단 `request.query_params` 로 ~

FYI, https://www.django-rest-framework.org/api-guide/requests/#query_params
```
request.query_params is a more correctly named synonym for request.GET.

For clarity inside your code, we recommend using request.query_params instead of the Django's standard request.GET.
```

Reply:
985c678 리뷰 반영했습니다.